docs-crawler/docs/docs/spiders/jax_spider.py

```diff
@@ -6,43 +12,88 @@
 
 
 class JaxSpider(CrawlSpider):
-    name = "jax"
-    version = "0.1.59"
-    allowed_domains = ['jax.readthedocs.io']
-    start_urls = ['https://jax.readthedocs.io/']
+    name = "jax"  # Name of web crawler
+    version = "0.1.59"  # Version of JAX module to crawl
+    allowed_domains = ['jax.readthedocs.io']  # Crawl only links from JAX
+    start_urls = ['https://jax.readthedocs.io/']  # Base URL for crawling
+
+    # Regex rules for compiling a string to a Regex object.
```

Comment:
It would be great to get a comment here explaining what this regex does, you can see an example here: https://github.com/OpenMined/Threepio/pull/100

It would be even better to refactor this so this regex isn't duplicated :)

Reply:
Thanks for the suggestion, I have added the documentation for the regex line. 
In regards to refactoring the line, I observed that not all the Scrapers have the same regex Instruction so I left it not to break the code.